<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerAxesModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerAxesModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="axes3DCollapsibleBtn" native="true">
     <property name="text" stdset="0">
      <string>Axes 3D</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QFormLayout" name="node3DFormLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="viewNode3DLabel">
          <property name="text">
           <string>View:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="viewNode3DSelector">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="showHidden" stdset="0">
           <bool>true</bool>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="renameEnabled" stdset="0">
           <bool>true</bool>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLViewNode</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="node3DLabel">
          <property name="text">
           <string>Node:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="qMRMLNodeComboBox" name="node3DSelector">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLDisplayableNode</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ticksHLayout">
        <item>
         <widget class="QCheckBox" name="xTicks3DCheckBox">
          <property name="text">
           <string>draw X-ticks</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="yTicks3DCheckBox">
          <property name="text">
           <string>draw Y-ticks</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="zTicks3DCheckBox">
          <property name="text">
           <string>draw Z-ticks</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="gridHLayout">
        <item>
         <widget class="QCheckBox" name="xMinorTicks3DCheckBox">
          <property name="text">
           <string>draw minor X-ticks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="yMinorTicks3DCheckBox">
          <property name="text">
           <string>draw minor Y-ticks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="zMinorTicks3DCheckBox">
          <property name="text">
           <string>draw minor Z-ticks</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="minorTicksHLayout">
        <item>
         <widget class="QCheckBox" name="xGrid3DCheckBox">
          <property name="text">
           <string>draw X-grid lines</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="yGrid3DCheckBox">
          <property name="text">
           <string>draw Y-grid lines</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="zGrid3DCheckBox">
          <property name="text">
           <string>draw Z-grid lines</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="labelVisibilityHLayout">
        <item>
         <widget class="QCheckBox" name="xLabel3DCheckBox">
          <property name="text">
           <string>show X-label</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="yLabel3DCheckBox">
          <property name="text">
           <string>show Y-label</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="zLabel3DCheckBox">
          <property name="text">
           <string>show Z-label</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="axes3hLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="updateAxes3DBtn">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QPushButton" name="showAxes3DBtn">
          <property name="text">
           <string>Show Axes 3D</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="axes2DCollapsibleBtn" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text" stdset="0">
      <string>Axes 2D</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="node2DFormLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="sliceNode2DLabel">
          <property name="text">
           <string>Slice:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="qMRMLNodeComboBox" name="sliceNode2DSelector">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLSliceNode</string>
           </stringlist>
          </property>
          <property name="showHidden" stdset="0">
           <bool>true</bool>
          </property>
          <property name="addEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="removeEnabled" stdset="0">
           <bool>false</bool>
          </property>
          <property name="renameEnabled" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="node2DLabel">
          <property name="text">
           <string>Node:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="qMRMLNodeComboBox" name="node2DSelector">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="nodeTypes" stdset="0">
           <stringlist>
            <string>vtkMRMLDisplayableNode</string>
           </stringlist>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="axes2DVisibilityHLayout">
        <item>
         <widget class="QCheckBox" name="xAxis2DCheckBox">
          <property name="text">
           <string>draw X-axis</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="yAxis2DCheckBox">
          <property name="text">
           <string>draw Y-axis</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="zAxis2DCheckBox">
          <property name="text">
           <string>draw Z-axis</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="axes2DLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="updateAxes2DBtn">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="showAxes2DBtn">
          <property name="text">
           <string>Show Axes 2D</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QComboBox</extends>
   <header>qMRMLNodeComboBox.h</header>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerAxesModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>viewNode3DSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>3</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerAxesModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>node3DSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>6</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerAxesModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>sliceNode2DSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>3</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerAxesModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>node2DSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>5</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
 </slots>
</ui>
